Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Use the official ROS 2 humble base image
+FROM ros:humble
+
+# Set the working directory
+WORKDIR /ros2_workspace/src
+
+COPY . .
```

Comment:
Please consider a more aware COPY than everything and only copy what is needed. This breaks the build cache, and you should have a .dockerignore file if you want to do this.

Source can be brought in through volume when the container is run, or through bind mount for each RUN step as needed.

Reply:
Fixed in https://github.com/avcuenes/grid_map_geo/pull/1